fetcher-pipeline/gl_repo_urls.txt

```diff
@@ -6,7 +6,7 @@ https://content.bibletranslationtools.org/WA-Catalog/pt-br_ulb
 https://content.bibletranslationtools.org/WA-Catalog/my_ulb
 https://content.bibletranslationtools.org/WA-Catalog/ceb_ulb
 https://content.bibletranslationtools.org/WA-Catalog/zh_ulb
-https://content.bibletranslationtools.org/WA-Catalog/en_ulb
```

Comment:
@danparisd it may need manual `rm` the en_ulb folder in the docker volume of _ORATURE_REPO_DIR_ so that the new repo url can be cloned with the same folder name